Rakefile

```diff
@@ -12,3 +12,10 @@ task :default do
 end
 
 PuppetLint.configuration.send('disable_80chars')
+
+
+begin
+  require 'puppet_blacksmith/rake_tasks'
+rescue LoadError
+end
+
```

Comment:
do you wanna copy the the thing i have in extlib, so we can release through travis when we tag?